index.html

```diff
@@ -15,26 +13,201 @@
 </head>
 
 <body>
-    <nav>
-      <i class='bx bx-menu bx-sm'></i>
-    </nav>
+  <nav>
+    <i class='bx bx-menu bx-sm'></i>
+  </nav>
   <section class="headline">
     <div class="container">
       <h1>Hey there. <br>
         I’m Tsige.</h1>
-      <h2 class="green">I’m a software developer! </h2></div>
-      <p>I can help you build any software as your need
-        including DesktopApp ,Mobile feature or website.
-        For more Please Look through
-        some of my work and experience that I have!</p><br>
-      <ul>
-        <li><a href="https://twitter.com/TsigeWo"><img src="icons/twitter.svg"></a></li>
-        <li><a href="https://www.linkedin.com/in/tsige-wossne-4782a2241/"><img src="icons/Linkedin icon.svg"></i></a></li>
-        <li><a href=""><img src="icons/medium.svg"></i></a></li>
-        <li><a href="https://github.com/tsigewossne"><img src="icons/github.svg"></a></li>
-        <li><a href=""><img src="icons/angellist.svg"></a></li>
-      </ul>
+      <h2 class="green">I’m a software developer! </h2>
     </div>
-      </section>
+    <p>I can help you build any software as your need
+      including DesktopApp ,Mobile feature or website.
+      For more Please Look through
+      some of my work and experience that I have!</p><br>
+    <div><ul>
+      <li><a href="https://twitter.com/TsigeWo"><img src="icons/twitter.svg"></a></li>
+      <li><a href="https://www.linkedin.com/in/tsige-wossne-4782a2241/"><img src="icons/Linkedin icon.svg"></i></a></li>
+      <li><a href=""><img src="icons/medium.svg"></i></a></li>
+      <li><a href="https://github.com/tsigewossne"><img src="icons/github.svg"></a></li>
+      <li><a href=""><img src="icons/angellist.svg"></a></li>
+    </ul></div>
+  </section>
+  <section class="next"><img src="icons/Disabled.svg" alt="next"></section>
+    <section class="works">
+      <h2>My Recent works</h2>
+      <h2>
+        <hr>
+      </h2>
+      <div class="container1">
+        <div class="img-placeholder">
+        </div>
+        <div class="text">
+          <h2>Multi-Post Stories Gain+Glory</h2>
+          <ul>
+            <li>Ruby on Rails</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+            <li>HTML</li>
```

Comment:
- [ ] I know you did hard work but there is a small change you need to work as the list `Ruby on Rails` the Rails is on a different line than in the design. I would suggest you please fix this issue. For this small change, I won't stop you from moving forward But make sure you implement this.
- [ ] Please Implement to all the list.
### Your design
![his](https://github.com/tsigewossne/portfolio/assets/81727988/a6612cbb-29cc-4365-9c12-0a13a0a3d3e5)